src/components/Landing/CallToAction.tsx

```diff
@@ -123,20 +123,21 @@ const CallToAction = () => {
 							<h1 className="landing-item-header">
 								For Property Managers
 							</h1>
-							<div className="">
-								<div className="">
+							<div className="landing-item-body">
+								<div className="img-cover">
 									<img
 										src={PropertyManager}
 										alt=""
 										className=""
 									/>
 								</div>
 								<div className="">
-									<ul>
-										<h2>
-											Improve efficiency and productivity
-											with our technology
-										</h2>
+									<h2>
+										Improve efficiency and productivity
+										with our technology
+									</h2>
+									<ul className="icon-list">
```

Comment:
you can do the items-start in this icon-list class

Reply:
Is this normal css class or a tailwindcss class

Reply:
It's plain css